src/debug/debug.cpp

```diff
@@ -2414,7 +2435,10 @@ static void OutputVecTable(char* filename) {
 		DEBUG_ShowMsg("DEBUG: Output of interrupt vector table failed.\n");
 		return;
 	}
-
+	DEBUG_ShowMsg("DEBUG: Interrupt vector table file '%s' created.\n",
+	              std_fs::absolute(filename).string().c_str()
+	              );
```

Comment:
After all implementation changes are done, fix the formatting please. Here's quick explanation and link to MSVC extension (or our script, whichever solution you prefer): https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#using-clang-format

Reply:
I use the MSVC plugin in VS when I rarely load up VS, but using Clang format directly via Vim commands is my preferred approach.

https://github.com/rhysd/vim-clang-format

You really want to be able to use the formatter on blocks only (e.g. when changing a 5-line function is a 3000-line legacy code, it would be obscene to reformat the whole file...)

Try using VS Code, it should have nice plugin with sane Clang Format support.

Reply:
> > > Just a single request, and please amend the commit message to follow our required style.
> > 
> > 
> > i did
> > git commit --amend and changed the text in both branches but can't see the change in this pull request
> 
> You need to force push and you should see them.

already did, there is no local change after running the script, git push --force gives 'Everything up-to-date'

Reply:
.\scripts\format-commit.sh does not change anything and `git force --push` is telling me `Everything up-to-date`

Reply:
@LowLevelMahn I don't know what's your output of `format-commit.sh` script and if you even have bash installed in your Windows machine…

Just manually fix the obvious formatting issue in line 2440 (move ending parenthesis to the previous line).

Reply:
im using git-bash or msys2 bash but format-commit.sh gives no change and i used that script in all my former PRs without a problem

normarly do the code change, commit, run the format-commit-script, check the output and then git push --force - everything on commandline